protobuf/model_config.proto

```diff
@@ -1869,6 +1869,81 @@ message ModelResponseCache
   bool enable = 1;
 }
 
+//@@
+//@@  .. cpp:var:: message ModelMetrics
+//@@
+//@@     The metrics setting of this model. Consider reusing this message body
+//@@     for backend metric custom configuration.
+//@@
+message ModelMetrics
+{
+  //@@
+  //@@  .. cpp:var:: message MetricControl
+  //@@
+  //@@     Override metrics settings of this model.
+  //@@
+  message MetricControl
+  {
+    //@@
+    //@@  .. cpp:var:: message MetricIdentifier
+    //@@
+    //@@     Specify metrics to be overridden with metric_option.
+    //@@
+    message MetricIdentifier
+    {
+      //@@  .. cpp:var:: string family
+      //@@
+      //@@     The name of the metric family to override with the custom value.
+      //@@     Check the list of supported metric families
+      //@@
+      // https://github.com/triton-inference-server/server/blob/main/docs/user_guide/metrics.md#histograms
+      //@@
+      string family = 1;
+    }
+
+    //@@  .. cpp:var:: message HistogramOptions
+    //@@
+    //@@     Histogram metrics options.
+    //@@
+    message HistogramOptions
+    {
+      //@@  .. cpp:var:: double buckets (repeated)
+      //@@
+      //@@     Repeated double type in ascending order for histogram bucket
+      //@@     boundaries. For example: [ -5.0, -2, 0, 3.5, 5 ].
```

Comment:
You should mention the openness of the boundary. From the testing it appears that for the given boundaries, you add `+Inf` in addition to the user provided boundary, also `-Inf` is not added which is asymmetric.

Reply:
Updated the comment